synapseclient/client.py

```diff
@@ -5129,22 +5130,27 @@ def _waitForAsync(self, uri, request, endpoint=None):
         # https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/asynch/AsynchronousJobStatus.html
         sleep = self.table_query_sleep
         start_time = time.time()
-        lastMessage, lastProgress, lastTotal, progressed = "", 0, 1, False
+        lastMessage, lastProgress, lastTotal = "", 0, 1
```

Comment:
Is this an opportunity to switch some variables over to camel-case? or are you just leaving all of the older logic alone with respect to formatting/style?

Reply:
For this PR, my focus was solely on ensuring that things use tqdm and the logger.

This code will eventually need to be patched, but a significant portion of it will also be deprecated or removed. I have a ticket to go through and deprecate everything that has a replacement, as well as to take stock of the items that don’t.